experimenter/experimenter/targeting/constants.py

```diff
@@ -3487,6 +3499,47 @@ def __post_init__(self):
     application_choice_names=(Application.DESKTOP.name,),
 )
 
+ACCEPTED_TOU_BEFORE_DEC_9_2025 = NimbusTargetingConfig(
+    name="Accepted TOU before Dec 9, 2025",
+    slug="accepted_tou_before_dec_9_2025",
+    description=(
+        "User accepted TOU before Dec 9, 2025 ",
+        "(excludes Linux)"
+    ),
```

Comment:
```suggestion
    description=(
        "User accepted TOU before Dec 9, 2025 "
        "(excludes Linux)"
    ),
```

this is presently a tuple of two strings